versions.cfg

```diff
@@ -1,45 +1,4 @@
 [versions]
 # pin according to requirements.
-setuptools = 33.1.1
-zc.buildout = 2.8.0
-
-# development dependencies (tools and pre commit hook)
-# most recent see bobtemplates.plone
-PyYAML = 3.12
-argh = 0.26.2
-args = 0.1.0
-check-manifest = 0.35
-clint = 0.5.1
-colorama = 0.3.7
-configparser = 3.5.0
-coverage = 3.7.1
-createcoverage = 1.5
-enum34 = 1.1.6
-flake8 = 3.3.0
-flake8-blind-except = 0.1.1
-flake8-coding = 1.3.0
-flake8-debugger = 1.4.0
-flake8-deprecated = 1.1
-flake8-isort = 2.1.3
-flake8-pep3101 = 1.0
-flake8-plone-api = 1.2
-flake8-plone-hasattr = 0.1
-flake8-polyfill = 1.0.1
-flake8-print = 2.0.2
-flake8-quotes = 0.9.0
-flake8-string-format = 0.2.3
-flake8-todo = 0.7
-isort = 4.2.5
-mccabe = 0.6.1
-pathtools = 0.1.2
-pkginfo = 1.4.1
-plone.recipe.codeanalysis = 2.2
-plone.testing = 5.0.0
-pycodestyle = 2.3.1
-pyflakes = 1.5.0
-requests-toolbelt = 0.7.1
-testfixtures = 4.13.4
-twine = 1.8.1
-watchdog = 0.8.3
-z3c.jbot = 0.7.2
-zest.releaser = 6.9
\ No newline at end of file
+setuptools =
+zc.buildout =
```

Comment:
These ones should be the same as requirements.txt.

Reply:
Yes.